docs/modules/ROOT/pages/references/slo-parameters.adoc

```diff
@@ -0,0 +1,31 @@
+= AppCat SLO Parameters
+
+The parent key for all the following parameters is `appcat.slos`.
+
+== `enabled`
+[horizontal]
+type:: bool
+default:: `true`
+
+If the SLOs should be enabled in general.
+
+== `namespace`
+[horizontal]
+type:: string
+default:: `appcat-slos`
+
+A namespace where all SLO related resources will be deployed to, including the SLI Exporter.
+The namespace will be created and managed by this component.
+
+== `sli_exporter.resources`
+[horizontal]
+type:: dict
+
+The resource requests and limits for the SLI Exporter.
+
+== `sli_exporter.kustomize_input`
+[horizontal]
+type:: dict
+
+The input passed to the Kustomize renderer.
```

Comment:
I don't understand why this is necessary, can you explain the necessity of this parameter here? Why not use the namespace from the parent object directly?

Reply:
This just exposes more option to configure kustomize, similarly to how we usually expose the complete helm values. This is what is recommended by https://syn.tools/syn/explanations/commodore-components/kustomizations.html